workflow/controller/operator.go

```diff
@@ -181,6 +181,49 @@ func newWorkflowOperationCtx(wf *wfv1.Workflow, wfc *WorkflowController) *wfOper
 	return &woc
 }
 
+// Determines if this workflow is definitely after the introduction of
```

Comment:
Can you make this a [godoc](https://go.dev/doc/comment) comment (e.g. start it with the function name) please.